src/main/java/uk/gov/hmcts/darts/common/entity/Courthouse.java

```diff
@@ -20,8 +23,8 @@ public class Courthouse {
 
     @Id
     @Column(name = "moj_crt_id")
-    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "courthouse_gen")
-    @SequenceGenerator(name = "courthouse_gen", sequenceName = "moj_crt_seq", allocationSize = 1)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "moj_cth_gen")
```

Comment:
should the sequence generator not match the name of the primary key? i.e. moj_crt_gen ?